### PR TITLE
DB_WEEK#2  PR Added

### PR DESCRIPTION
**Pull request by Elia-Git**
Homework DB_W2

### CODE REVIEW

File: databases/week2/part2.sql [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+-- Create an index on name column of the student table
+USE school_database;
+CREATE INDEX index_name ON student(name);
+
+-- Add a new column to the class table named status
+-- with enum(not-started, ongoing, finished)
+
+ALTER TABLE class
+ add STATUS ENUM("not-started", "ongoing", "finished"
+) NOT null ;
```
Comment (babakbandpey) [inline databases/week2/part2.sql:18]: There is a need for a default value when a new column is added. Research it.

File: databases/week2/part3.sql [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  JOIN status ON task.status_id = status.id
+WHERE user.name='Donald Duck' AND status.id =1;
+
+-- Get all the tasks for 'Maryrose Meadows' that were created in september (hint: month(created)=month_number)
+SELECT title, month(created), user.name Created_by
+FROM task
+  JOIN user
+  ON task.id=user.id
+WHERE user.name = 'Maryrose Meadows' AND task.created LIKE "%-09-%";
+
```
Comment (babakbandpey) [inline databases/week2/part3.sql:22]: it will be more clear to use `month(task.created) = 9`

File: databases/week2/part4.sql [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+(25) NOT NULL,
+  `Last_name` VARCHAR
+(25) NOT NULL,
+  `User_name` VARCHAR
+(25) NOT NULL,
+  `Password` VARCHAR
+(25) NOT NULL,
+  `Phone` VARCHAR
+(45) NOT NULL,
+  PRIMARY KEY
```
Comment (babakbandpey) [inline databases/week2/part4.sql:80]: Primary key must not be the email
What will happen if the user changes email address?
Please add an id int auto increment

File: databases/week2/part4.sql [hunk 4]
```diff
@@ -0,0 +1,160 @@
...
+ENGINE = InnoDB;
+
+
+-- -----------------------------------------------------
+-- Table `Job_interview`.`User_has_Interview`
+-- -----------------------------------------------------
+CREATE TABLE
+IF NOT EXISTS `Job_interview`.`User_has_Interview`
+(
+  `User_Email` INT NOT NULL,
```
Comment (babakbandpey) [inline databases/week2/part4.sql:91]: this must be the id from the user table

File: databases/week2/part4.sql [hunk 5]
```diff
@@ -0,0 +1,160 @@
...
+ENGINE = InnoDB;
+
+
+-- -----------------------------------------------------
+-- Table `Job_interview`.`User_has_Appointment`
+-- -----------------------------------------------------
+CREATE TABLE
+IF NOT EXISTS `Job_interview`.`User_has_Appointment`
+(
+  `User_Email` INT NOT NULL,
```
Comment (babakbandpey) [inline databases/week2/part4.sql:126]: change it to user_id and use the id from the user table

Comment (Elia-Git) [reply databases/week2/part4.sql]: Fixed !

File: databases/week2/part4.sql (hunk 4 above)
Comment (Elia-Git) [reply databases/week2/part4.sql]: Fixed !

File: databases/week2/part4.sql (hunk 3 above)
Comment (Elia-Git) [reply databases/week2/part4.sql:85]: Implemeted!

File: databases/week2/part3.sql (hunk 2 above)
Comment (Elia-Git) [reply databases/week2/part3.sql:22]: Looks better now,thank you!

**Comment by Elia-Git** [comment]
Merged DB_hw_w2

**Pull request merged**